src/App.js

```diff
@@ -1,16 +1,45 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+import { useState} from 'react';
 
 const App = () => {
+  // I need an array that has all of my state
+  // const initialCopy = chatMessages.map(chat=> {
+  //   return {...chat,
+  const [initialCopy, setChatEntries] = useState(chatMessages);
+  const updateIsLiked = (id) => {
+    const newChatEntries = [];
+    for (const chat of initialCopy) {
+      if (chat.id !== id) {
+        newChatEntries.push(chat);
+      } else {
+        const newChatMessage = {
```

Comment:
Make sure you're keeping track of your indentation levels! Lines 18-26 should all be indented one more level. Good code style is important for readability and maintainability. 🧹 😌